tests/functional/UseCase/Workspace/CreateDropWorkspaceTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\FunctionalTests\UseCase\Workspace;
+
+use Keboola\StorageDriver\Command\Project\CreateProjectResponse;
+use Keboola\StorageDriver\Command\Workspace\DropWorkspaceCommand;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\FunctionalTests\BaseCase;
+use Keboola\StorageDriver\Teradata\Handler\Workspace\Drop\DropWorkspaceHandler;
+
+class CreateDropWorkspaceTest extends BaseCase
+{
+    protected GenericBackendCredentials $projectCredentials;
+
+    protected CreateProjectResponse $projectResponse;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->cleanTestProject();
+        [$credentials, $response] = $this->createTestProject();
+        $this->projectCredentials = $credentials;
+        $this->projectResponse = $response;
+    }
+
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+        $this->cleanTestProject();
+    }
+
+    public function testCreateDropWorkspace(): void
+    {
+        [$response, $db] = $this->createTestWorkspace($this->projectCredentials, $this->projectResponse);
```

Comment:
Pridat jeste check, ze existuje role, jde vytvorit tabulka, user ma prislusna prava...

Reply:
jj že ide vytvořit tabulka a třeba view pod těma WS credentials a pak že ten drop úspěšně projde. Pak sa následně vyzkůšá Load a Unload, ale to sa udělá až v jiném PR.

Reply:
do testu pridany granty